packages/ui-eslint-config/package.json

```diff
@@ -21,7 +21,6 @@
     "@typescript-eslint/parser": "^4.22.0",
     "eslint-config-prettier": "^8",
     "eslint-plugin-compat": "^3",
-    "eslint-plugin-ejs": "^0.0.2",
```

Comment:
I've removed .ejs file linting. We have like 5 ejs files in the repo (and most of them in the deprecated template apps); this plugin was not updated since 5 years and has a rarely occuring crash that takes hours to debug (was hit by it during this upgrade)